projects/components/src/utils/test/datagrid/datagrid.wo.ts

```diff
@@ -37,11 +35,11 @@ export class ClrDatagridWidgetObject extends WidgetObject<ClrDatagrid> {
 
     /**
      * Retrieves if the cell will clip text
-     * @param row 0-based index of row
      * @param column 0-based index of column
      */
-    cellWillClip(row: number, column: number): boolean {
-        return this.getCell(row, column).providerTokens.some(token => token.name === 'ShowClippedTextDirective');
+    columnHasClippedTextDirective(column: number): boolean {
```

Comment:
Since we are always adding the directive, this name doesn't make sense anymore...